cloudbuild.yaml

```diff
@@ -0,0 +1,28 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+timeout: 3600s
+
+options:
+  substitution_option: ALLOW_LOOSE
+
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f'
+    entrypoint: 'bash'
```

Comment:
Can't we just use make as the entrypoint? See eg https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/cloudbuild.yaml.

Reply:
sounds good, will update